### DISCUSSION TITLE
Error - Exception in key sending

### DISCUSSION THREAD

**Discussion by mediahope** [rsjaffe/MIDI2LR#755] [Problems]
Often, after starting lightroom, the keyboard commands do not work. Always by accident. Sometimes everything is fine and works.

Error - Exception in key sending function for key: "w" (or another). Exception: .
Error - Exception

![image](https://github.com/rsjaffe/MIDI2LR/assets/13403919/1daaf01d-f530-497b-ac60-990bc452a15a)

**Comment by mediahope** [comment]
This error is related to the language layout.

**To fix:**

1. Switch the language to English
2. Close the extension
3. Open the extension

However, I hope that the developer will be able to fix this inconvenient detail..

**Comment by antzpantz** [comment]
Which language were you using? And were you changing the Regional Settings
in Windows Settings?

**Reply by mediahope** [reply]
The keyboard layout is Russian, when switching to English and restarting - everything works as it should.
I didn't change anything in the regional settings, 2 languages are available - Russian and English.
Thank you!
